client/src/components/generic/ConfirmationDialog/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,80 @@
+import Button from "../FigmaButtons/FigmaButton"
+type dialogVariants = "default"
+
+export interface IDialogProps {
+  title: string
+  text: string
+  variant?: dialogVariants
+  left: string
```

Comment:
what does `left` and `right` mean? make it more clear or add jsdoc explaining it?

Reply:
changed it